commons/commons-util/src/test/kotlin/ebi/ac/uk/io/FileUtilsTest.kt

```diff
@@ -246,11 +246,24 @@ internal class FileUtilsTest(private val temporaryFolder: TemporaryFolder) {
         }
 
         @Test
-        fun size() {
+        fun `file size`() {
             val file = temporaryFolder.createFile("size-test.txt", "a test text")
             assertThat(FileUtils.size(file)).isEqualTo(11L)
         }
 
+        @Test
+        fun `directory size`() {
+            val directory = temporaryFolder.createDirectory("dir-size-test")
+
+            temporaryFolder.createDirectory("dir-size-test/level-1")
+            temporaryFolder.createDirectory("dir-size-test/level-1/level-2")
+            temporaryFolder.createFile("dir-size-test/file1.txt", "test 1")
+            temporaryFolder.createFile("dir-size-test/level-1/file2.txt", "test 2")
+            temporaryFolder.createFile("dir-size-test/level-1/level-2/file3.txt", "test 3")
+
+            assertThat(FileUtils.size(directory)).isEqualTo(18L)
```

Comment:
I am wondering if we shoud do something like 
```
        @Test
        fun `directory size`() {
            val directory = temporaryFolder.createDirectory("dir-size-test")

            temporaryFolder.createDirectory("dir-size-test/level-1")
            temporaryFolder.createDirectory("dir-size-test/level-1/level-2")
            val f1 = temporaryFolder.createFile("dir-size-test/file1.txt", "test 1")
            val f2 = temporaryFolder.createFile("dir-size-test/level-1/file2.txt", "test 2")
            val f3 = temporaryFolder.createFile("dir-size-test/level-1/level-2/file3.txt", "test 3")

            assertThat(FileUtils.size(directory)).isEqualTo(f1.size() + f2.size() + f3.size())
        }
```

I am aware we would be  asserting function `FileUtils.size` using the function `size`  but as it is right now it looks super hard to be to guess expected size is 18.